src/arch/riscv/RiscvISA.py

```diff
@@ -52,9 +55,19 @@ class RiscvISA(BaseISA):
     cxx_class = "gem5::RiscvISA::ISA"
     cxx_header = "arch/riscv/isa.hh"
 
+    system = Param.System(Parent.any, "System this ISA object belongs to")
+
     check_alignment = Param.Bool(
         True, "whether to check memory access alignment"
     )
+
     riscv_type = Param.RiscvType("RV64", "RV32 or RV64")
 
     enable_rvv = Param.Bool(True, "Enable vector extension")
+
+    riscv_vlen_se = Param.RiscvVectorLength(
```

Comment:
I'm a bit confused on the variable name. I would have simply called it `vlen` and put in the description "Length of each vector register in bits. VLEN in Ch. 2 of RISC-V vector spec"

I know it's a bit nit-picky, but we're going to be stuck with these names forever :)